### PR TITLE
PDA-192 added save settings wrapper for local dev

### PR DESCRIPTION
**Pull request by rimselis**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/App/App.js
Comment (gReis89) [inline widgets/src/components/App/App.js]: can we remove this line now?

File: widgets/src/components/common/WidgetConfig.js
Comment (gReis89) [inline widgets/src/components/common/WidgetConfig.js:47]: why declaring a function here if you are calling it right after its declaration?

File: widgets/src/components/common/WidgetConfig.js
Comment (gReis89) [inline widgets/src/components/common/WidgetConfig.js:41]: do we need this?

File: widgets/src/components/common/WidgetConfig.js
Comment (gReis89) [inline widgets/src/components/common/WidgetConfig.js:14]: can't it be a boolean instead?

File: widgets/src/components/App/App.js
Comment (rimselis) [reply widgets/src/components/App/App.js]: Left there to indicate that that is possible (otherwise devs will have to investigate WidgetConfig)

File: widgets/src/components/common/WidgetConfig.js
Comment (rimselis) [reply widgets/src/components/common/WidgetConfig.js:47]: Using async directly in the useEffect function isn't allowed (`useEffect function must return a cleanup function or nothing. Promises and useEffect(async () => ...) are not supported, but you can call an async function inside an effect`)

File: widgets/src/components/common/WidgetConfig.js
Comment (rimselis) [reply widgets/src/components/common/WidgetConfig.js:41]: setTimeout part? Simplest way to give feedback to developer without connecting notifications or adding alerts

File: widgets/src/components/common/WidgetConfig.js
Comment (rimselis) [reply widgets/src/components/common/WidgetConfig.js:14]: It shows different icon in case there is an error. It's just simplest way to give feedback to developer without connecting notifications or adding alerts - does not affect development that much

**Review by gReis89 — DISMISSED** [review]
changes looks good!

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/114#pullrequestreview-401861735" title="Approved">gReis89</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

**Review by gReis89 — APPROVED** [review]
looks good

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/114#pullrequestreview-401890685" title="Approved">gReis89</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Pull request merged**
